src/PlaywrightSharp.ProtocolTypesGenerator/StandardProtocolTypesGeneratorBase.cs

```diff
@@ -77,38 +87,46 @@ private void PrepareArrayTypes(ProtocolDomainsContainer response)
             _knownTypes["StringIndex"] = "int";
         }
 
-        private void GenerateTypes(StringBuilder builder, ProtocolDomain domain)
+        private void GenerateEnums(StringBuilder builder, ProtocolDomain domain)
         {
             if (domain.Types == null)
             {
                 return;
             }
 
-            foreach (var type in domain.Types)
+            // Generate enums
+            foreach (var type in domain.Types.Where(t => t.Enum != null))
             {
-                if (type.Enum != null)
-                {
-                    _enums.Add(type.Id);
+                _enums.Add($"{domain.Domain}.{type.Id}");
```

Comment:
There are references between domains. For instance, inside `Fetch` we have `Network.ResourceType`.